Dockerfile

```diff
@@ -47,11 +47,17 @@ RUN set -ex \
         rsync \
         netcat \
         locales \
+        python3 \
+        python3-pip \
+        libpython3-dev \
+    && ln -s /usr/bin/python3 /usr/bin/python \
+    && ln -s /usr/bin/pip3 /usr/bin/pip \
```

Comment:
Looking at how the previous image did this, didn't seem them do this symlinking https://github.com/docker-library/python/blob/4df129649329186fb42c9c33215b41d82bc0e58e/3.6/stretch/slim/Dockerfile

I see them symlinking python3 to python, but not pip3 to pip. Is it done somewhere else?

Reply:
I think pip isn't in that list because that dockerfile installs pip in a really bizarre way using its own get-pip.py, but I think for what we care about using the version of pip in the ubuntu repository should be fine.